app/assets/javascripts/components/bulkUploadImport.jsx

```diff
@@ -186,12 +193,16 @@ class BulkUploadImport extends React.Component {
         createdSampleIds: response.data.sample_ids
       });
       setTimeout(() => {
+        that.setState({
+          submitting: false
+        });
         that.gotoPage(`/?ids=${that.state.createdSampleIds.join(',')}`);
       }, 5000)
```

Comment:
maybe we should make the delay just 200ms instead. otherwise people will click on the button again